spec.html

```diff
@@ -31547,7 +31547,7 @@ <h1>%RegExpStringIteratorPrototype%.next ( )</h1>
 
         <emu-clause id="sec-%regexpstringiteratorprototype%-@@tostringtag">
           <h1>%RegExpStringIteratorPrototype% [ @@toStringTag ]</h1>
-          <p>The initial value of the _@@toStringTag_ property is the String value *"RegExp String Iterator"*.</p>
+          <p>The initial value of the _@@toStringTag_ property is the String value `"RegExp String Iterator"`.</p>
```

Comment:
The italics on @@toStringTag here seem to be a mistake too.

Reply:
@rkirsling indeed it is. I wasn't looking for anything outside of the cases that I was trying to correct, but I don't mind making another pass to fix any incorrect formatting that matches this pattern.